### ISSUE TITLE
[New Feature] Releasing Tiberium on death

### ISSUE THREAD

**Issue by askhati** [Vinifera-Developers/Vinifera#580]
### Description:
Currently, when an unit has the tag "TiberiumHeal=yes", two things happen:
 - the unit will heal when standing in tiberium fields
 - ...and will ALSO release tiberium on death
 
 For balancing and unit design purposes, it would be great if these two functions could be unbundled into separate tags.

### Possible Implementation:
Create the following two new tags:
  - TiberiumHealer= y/n (heal in tib; nothing else)
  - TiberiumOnDeath= y/n (release tib on death)
  
  That way, the vanilla "TiberiumHeal=" can be used on units that have both functions; for units that need only one of the two abilities, use the appropriate new tag.

**Comment by Rampastring** [comment]
I think the almost-duplicate keys could end up being confusing. As an alternative, we could only add the `TiberiumOnDeath` key. The functionality that a unit would have with only `TiberiumHealer=yes` could be achieved with `TiberiumHeal=yes` and `TiberiumOnDeath=no`.

**Comment by ErastusMercy** [comment]
I agree with Rampa, I think the `TiberiumOnDeath` could simply act as an override for that element of the `TiberiumHeal` behaviour.

**Comment by CCHyper** [comment]
Just checked this out, looks pretty straightforward.

Is there anything else related to this before I go ahead and work on this?

**Comment by askhati** [comment]
As discussed on Discord, an update of the initial request would look like this:

 - only 1x new tag: "TiberiumOnDeath="
  - accepts **no/false** OR a **valid Tiberium type** from the Global list (Riparius, Arboreus, Vinifera, etc)
  - if no/false: use this to disable the tiberium-spawning effect of the existing WW tag "TiberiumHeal="
  - if a valid Tiberium type: spawn this Tiberium type on death; needs to interact with the existing WW "TiberiumHeal=" tag to potentially override the Tiberium type released by that (first) tag if it is different
